typescript-selenium/README.md

```diff
@@ -0,0 +1,24 @@
+## Requirements
+- node
+- "Chrome browser"
+
+## Default launch
+- npm install && npm test
+
+## Custom launch
+**Use environment variables (check default values in 'TestConstants.ts' file):**
```

Comment:
I suggest to only mentions the file `TestConstants.ts` here but remove the list of all options. Additionally, it would be nice to have some documentation of these options.